.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,10 @@
 ## PR Checklist
```

Comment:
This template needs to be updated to out latest. Use this one as a reference and update hyper link appropriately 
https://github.com/edgexfoundry/app-service-configurable/blob/master/.github/PULL_REQUEST_TEMPLATE.md